test/libyul/yulSyntaxTests/selfdestruct_pre_cancun.yul

```diff
@@ -0,0 +1,8 @@
+
+{
+	selfdestruct(0x02)
+}
+// ====
+// EVMVersion: <=shanghai
+// ----
+// Warning 1699: (4-16): "selfdestruct" has been deprecated. Note that, since the Cancun hard fork, the underlying opcode no longer deletes the code and data associated with an account and only transfers its Ether to the beneficiary, unless executed in the same transaction in which the contract was created (see EIP-6780). Any use in newly deployed contracts is strongly discouraged even if the new behavior is taken into account. Future changes to the EVM might further reduce the functionality of the opcode.
```

Comment:
If we're issuing the same warning on all forks, then it might make sense to change the phrasing to
```
Note that, starting from the Cancun hard fork, ....
```